frontend/components/site/SiteUsers.jsx

```diff
@@ -2,12 +2,20 @@ import React from 'react';
 import { SITE, USER } from '../../propTypes';
 import ButtonLink from '../ButtonLink';
 import siteActions from '../../actions/siteActions';
+import UserActionsTable from './UserActionsTable';
 
 import GitHubMark from '../GitHubMark';
 
 const isSiteOwner = (user, siteOwner) =>
   user.username.toLowerCase() === siteOwner.toLowerCase();
 
+const renderUserActions = (userActions) => {
+  if (!userActions.length) {
```

Comment:
I don't know what was going on, but I was getting `userActions` as `undefined` (and thus throwing an error here) when I first switched to this PR after working on something else. It might have been a blip from switching branches though.

Nonetheless, maybe a default value of `[]` should be set in `defaultProps`? Or, since this table is likely going to be moved in the future, maybe don't worry about it here for now.

Reply:
I think something must have been up with the state of my db, because looking at the user serializer, there's no way that should have been `undefined`